executor/aggregate.go

```diff
@@ -63,6 +66,7 @@ func newBaseHashAggWorker(ctx sessionctx.Context, finishCh <-chan struct{}, aggF
 type HashAggPartialWorker struct {
 	baseHashAggWorker
 
+	stats             *HashAggRuntimeStats
```

Comment:
How about put `	stats               *HashAggRuntimeStats` in `baseHashAggWorker `?